src/content/docs/pt-br/guides/upgrade-to/v4.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Atualize para o Astro v4
+description: Como atualizar seu projeto para a versão mais recente do Astro (v4.0).
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+Este guia vai te ajudar a migrar do Astro v3 para o Astro v4.
+
+Precisa atualizar um projeto antigo para a v3? Veja nosso [antigo guia de migração](/pt-br/guides/upgrade-to/v3/).
+
+Precisa ver a documentação da v3? Visite essa [versão antiga do site de documentação (snapshot v3.6 não mantido)](https://docs-git-v3-docs-unmaintained-astrodotbuild.vercel.app/).
+
+## Atualize o Astro
+
+Atualize a versão do Astro e de todas as integrações oficiais em seu projeto para a versão mais recente utilizando seu gerenciador de pacotes.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Atualiza o Astro e integrações oficiais juntos
+  npx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Atualiza o Astro e integrações oficiais juntos
+  pnpm dlx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Atualiza o Astro e integrações oficiais juntos
+  yarn dlx @astrojs/upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Você também pode [atualizar suas integrações do Astro manualmente](/pt-br/guides/integrations-guide/#upgrade-manual) se preferir, e você pode precisar atualizar outras dependências no seu projeto.
+
+:::note[É necessário continuar?]
+Após atualizar Astro para a versão mais recente, você pode não precisar fazer quaisquer mudanças ao seu projeto afinal!
+
+Porém, se você notar erros ou comportamentos inesperados, por favor veja abaixo o que mudou que pode precisar ser atualizado em seu projeto.
+:::
+
+Astro v4.0 inclui [mudanças potencialmente radicais](/pt-br/guides/upgrade-to/v4/#mudanças-radicais), assim como [remoção de funcionalidades anteriormente descontinuadas](/en/guides/upgrade-to/v4/#funcionalidades-previamente-descontinuadas-agora-removidas).
```

Comment:
```suggestion
Astro v4.0 inclui [mudanças potencialmente radicais](#mudanças-radicais), assim como [remoção de funcionalidades anteriormente descontinuadas](#funcionalidades-previamente-descontinuadas-agora-removidas).
```
Since these links are on the same page, this should work!

Reply:
Yeah, I though that too, but since the original includes the path in the links I kept them here as well

Reply:
Either way is fine, but notice that the second one was pointing to English, not within the same page.